tests/test_system.py

```diff
@@ -2,535 +2,64 @@
 import importlib
 import sys
 
-MODULES = sys.modules.keys()
+import pytest
+
+_MODULES = sys.modules.keys()
+
+_SYSTEM_MODULES = [
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (testing):** Consider adding a test case for dynamically loaded modules.

The current parametrization covers a predefined list of modules. It might be beneficial to also include a test case that ensures dynamically loaded or newly added modules are handled correctly. This could involve mocking or a setup that simulates the dynamic loading of modules.